src/core/optimization_container.jl

```diff
@@ -881,9 +880,9 @@ function _add_expression_container!(
     sparse = false,
 )
     if sparse
-        expr_container = sparse_container_spec(JuMP.AbstractJuMPScalar, axs...)
+        expr_container = sparse_container_spec(JuMP.GenericAffExpr, axs...)
```

Comment:
You need to be careful with this change since it can break when we need these to store `PGAE`. If you are unsure, let's just put Any or something super generic. This will be problematic until we can use `ParametricOptInterface`

Reply:
I think the super generic type that would work will be `JuMP.AbstractJuMPScalar`, So I'll leave as is